src/api/entitlements.js

```diff
@@ -82,6 +90,26 @@ export class Entitlements {
     };
   }
 
+  /**
+   * Returns true if the entitlements enable the current product
+   * with a cacheable entitlement.
+   * @return {boolean}
+   */
+  enablesThisWithCacheableEntitlements() {
+    const entitlement = this.getEntitlementForThis();
+    return !!entitlement && entitlement.source !== GOOGLE_METERING_SOURCE;
+  }
+
+  /**
+   * Returns true if the entitlements enable the current product
+   * with a Google metering entitlement.
```

Comment:
ditto - comment is not clear

Reply:
I updated this comment and added more details